synapse/tests/test_telepath.py

```diff
@@ -171,38 +166,37 @@ class TeleTest(s_t_utils.SynTest):
 
             await self.asyncraises(s_exc.BadUrl, s_telepath.openurl('noscheme/foo'))
 
-            prox = await s_telepath.openurl('tcp://127.0.0.1/foo', port=addr[1])
-            # Add an additional prox.fini handler.
-            prox.onfini(evt.set)
+            async with await s_telepath.openurl('tcp://127.0.0.1/foo', port=addr[1]) as prox:
```

Comment:
The reason for NOT using the proxy as a context manager in this case was to keep the proxy alive after the dmon's context manager exited and fini'd him (see the section beneath it). This indentation should be reverted.